I'm having a really hard time figuring out what is causing that error... where is the comparison that's resulting in that failure?

```
--- FAIL: ExamplePrintReplicationController (0.00s)
got:
CONTROLLER   CONTAINER(S)   IMAGE(S)    SELECTOR   REPLICAS
foo          foo            someimage   foo=bar    1
want:
CONTROLLER          CONTAINER(S)        IMAGE(S)            SELECTOR            REPLICAS
foo                 foo                 someimage           foo=bar             1
FAIL
FAIL    github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd       0.038s
```

--- a/pkg/kubectl/cmd/cmd_test.go
+++ b/pkg/kubectl/cmd/cmd_test.go
@@ -238,6 +238,6 @@ func ExamplePrintReplicationController() {
 		fmt.Printf("Unexpected error: %v", err)
 	}
 	// Output:
-	// CONTROLLER          CONTAINER(S)        IMAGE(S)            SELECTOR            REPLICAS
-	// foo                 foo                 someimage           foo=bar             1
+	// CONTROLLER   CONTAINER(S)   IMAGE(S)    SELECTOR   REPLICAS
+	// foo          foo            someimage   foo=bar    1
 }

--- a/pkg/kubectl/resource_printer.go
+++ b/pkg/kubectl/resource_printer.go
@@ -550,7 +550,7 @@ func printResourceQuotaList(list *api.ResourceQuotaList, w io.Writer) error {
 
 // PrintObj prints the obj in a human-friendly format according to the type of the obj.
 func (h *HumanReadablePrinter) PrintObj(obj runtime.Object, output io.Writer) error {
-	w := tabwriter.NewWriter(output, 20, 5, 3, ' ', 0)
+	w := tabwriter.NewWriter(output, 10, 4, 3, ' ', 0)
 	defer w.Flush()
 	t := reflect.TypeOf(obj)
 	if handler := h.handlerMap[t]; handler != nil {

